scripts/macpython-download-cache-and-build-module-wheels.sh

```diff
@@ -2,15 +2,43 @@
 
 # This module should be pull and run from an ITKModule root directory to generate the Mac python wheels of this module,
 # it is used by the .travis.yml file contained in ITKModuleTemplate: https://github.com/InsightSoftwareConsortium/ITKModuleTemplate
+# 
+# Variables to be set before calling the script:
+# - ITK_PACKAGE_VERSION: Tag for ITKPythonBuilds build archive to use
+# - ITKPYTHONPACKAGE_TAG: Tag for ITKPythonPackage build scripts to use.
+#     If ITKPYTHONPACKAGE_TAG is empty then the default scripts distributed
+#     with the ITKPythonBuilds archive will be used.
+# - ITKPYTHONPACKAGE_ORG: Github organization or user to use for ITKPythonPackage
+#     build script source. Default is InsightSoftwareConsortium.
+#     Ignored if ITKPYTHONPACKAGE_TAG is empty.
 
+# Install dependencies
 brew update
 brew install zstd aria2 gnu-tar doxygen ninja
 brew upgrade cmake
+
+# Fetch ITKPythonBuilds archive containing ITK build artifacts
 aria2c -c --file-allocation=none -o ITKPythonBuilds-macosx.tar.zst -s 10 -x 10 https://github.com/InsightSoftwareConsortium/ITKPythonBuilds/releases/download/${ITK_PACKAGE_VERSION:=v5.3.0}/ITKPythonBuilds-macosx.tar.zst
 unzstd --long=31 ITKPythonBuilds-macosx.tar.zst -o ITKPythonBuilds-macosx.tar
 PATH="/usr/local/opt/gnu-tar/libexec/gnubin:$PATH"
 tar xf ITKPythonBuilds-macosx.tar --checkpoint=10000 --checkpoint-action=dot
 rm ITKPythonBuilds-macosx.tar
+
+# Optional: Update build scripts
+if [[ -n ${ITKPYTHONPACKAGE_TAG} ]]; then
+  echo "Updating build scripts to ${ITKPYTHONPACKAGE_ORG:=InsightSoftwareConsortium}/ITKPythonPackage@${ITKPYTHONPACKAGE_TAG}"
+  git clone "https://github.com/${ITKPYTHONPACKAGE_ORG}/ITKPythonPackage.git" "IPP-tmp"
```

Comment:
Should this be:
```suggestion
  git clone "https://github.com/${ITKPYTHONPACKAGE_ORG:=InsightSoftwareConsortium}/ITKPythonPackage.git" "IPP-tmp"
```

Reply:
Good question, parameter assignment happens in line 29 with `${ITKPYTHONPACKAGE_ORG:=InsightSoftwareConsortium}` so that `ITKPYTHONPACKAGE_ORG` is already initialized when execution reaches line 30. We only need one `:=`.

Explanation at https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html